framework/SupervisedLearning/ARMA.py

```diff
@@ -279,11 +287,14 @@ def __trainLocal__(self,featureVals,targetVals):
         self.notZeroFilterMask = self._trainZeroRemoval(timeSeriesData,tol=self.zeroFilterTol) # where zeros or less than zeros are
         self.zeroFilterMask = np.logical_not(self.notZeroFilterMask) # where data are
       # if we're removing Fourier signal, do that now.
-      maskRes = timeSeriesData == timeSeriesData
 
+      maskRes = np.ones(len(timeSeriesData), dtype=bool)
```

Comment:
Can we rename "maskRes" to something that reflects that it's a mask for the peaks?